### ISSUE TITLE
Unlink URI from resource type

### ISSUE THREAD

**Issue by antonkomarev** [DavidDuwaer/Coloquent#70]
Right now `jsonApiType` is using for resource `type` and in URI builder. But in JSON:API specification there is no rule to keep them same.

Example:
I have resource with `type: "Truck"`, but it's listed in `GET /cars` endpoint.

I propose to have extra property which will be used in URI path. If it's value is undefined or null - then `jsonApiType` will be used as it's working now.

**Comment by DavidDuwaer** [comment]
Makes sense, good idea

**Comment by DavidDuwaer** [comment]
Would you have time to implement this?

**Comment by antonkomarev** [comment]
Nope, sorry. I'm not actually using your package. Just made a research of existing solutions and found this issue.
